docs/configuration.md

```diff
@@ -1192,6 +1192,14 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+    <td><code>spark.files.filterOutEmptySplit</code></td>
```

Comment:
I don't think I'd document this. It should be just a safety valve flag

Reply:
yea we can make this conf an internal conf.